omod/src/main/webapp/pages/editSection.gsp

```diff
@@ -220,10 +220,11 @@ ${ ui.includeFragment("uicommons", "validationMessages")}
             <div class="before-dataCanvas"></div>
             <div id="dataCanvas"></div>
             <div class="after-data-canvas"></div>
-            <div id="confirmationQuestion">
+            <div id="confirmationQuestion" class="focused"> confirm submission?
                 ${ui.message("registrationapp.confirm")}
```

Comment:
`${ui.message("registrationapp.confirm")}` is the code that makes the UI message appear on the screen. We don't hard-code text (like "confirm submission?"). I expect that this code will make a div that has contents like "confirm submission?Confirm", assuming the message for `registrationapp.confirm` is "Confirm".

Reply:
Same goes for the "Confirm" message below.

Reply:
@brandon thanks for the review am fixing it.